sections/GPU.tex

```diff
@@ -1,14 +1,21 @@
-\section{Graphics Processing Units}
-\label{sec:GPU}
-Dedicated Graphic Processing Units - GPUs - are very common in desktop and laptops. \citep{STEAMHW}
+\section{Computation architectures}
+\label{sec:comparch}
+Normally computations are performed on a computer's Central Processing Unit - a CPU.
+A CPU is often fastest for everyday task and calculations.
+The CPU consist of only a few cores which are highly optimized for sequential serial processing.\citep{whatisgpu}
+But in recent years developers and scientists seem to have developed at interest for performing calculations on Dedicated Graphic Processing Units.\citep{gpurise}
+This is a whole other architecture which normally only was used for graphics processing but with some problems it seems that one could utilise a GPU for General Purpose GPU computing - GPGPU, with an performance improvement over a CPU.
```

Comment:
Igen, hvad er normen?
This is a different architecture which originally was designed for graphics processing [...]
"but with some problems it seems that one could utilise" Split sætningen, det er forvirrende at det er en del af det ovenstående.
Some computational problems could utilise [...]